src/com/projectkorra/projectkorra/earthbending/RaiseEarth.java

```diff
@@ -104,7 +102,7 @@ private void setFields() {
 
 	private boolean canInstantiate() {
 		for (final Block block : this.affectedBlocks.keySet()) {
-			if (!this.isEarthbendable(block) || ALL_AFFECTED_BLOCKS.containsKey(block)) {
+			if (!this.isEarthbendable(block) || TempBlock.isTempBlock(block)) {
```

Comment:
there is already a check for tempblocks in the move earth code, so I dont think this is needed

Reply:
it doesn't hurt to have it